person.rb

```diff
@@ -1,7 +1,10 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :id, :name, :age, :parent_permission
 
   def initialize(age, parent_permission: true, name: 'Unknown')
+    super()
```

Comment:
| Error :red_circle: |
|-|
| ![image](https://github.com/Theodoraldo/OOP_school_library/assets/93039359/c6a905cb-92b9-4a42-b5e5-0b3e65b74715) |

- [ ] It would be nice if you make your initialize method arguments as you can see below :point_down: to prevent getting an error which we are getting you can see in the image above :arrow_up:

```ruby
def initialize(age, name = "Unknown", parent_permission: true)
```